src/protagonist/Portal/Pages/Index.cshtml

```diff
@@ -1,205 +1,124 @@
-              Assets can be arranged in IIIF Manifests for use in viewers and other software.
-              PDFs can optionally be converted to IIIF Manifests... and Manifest to PDF!
-            </p>
-            <div class="d-grid gap-2 d-md-flex justify-content-md-start">
-              <a asp-page="/Signup" class="btn btn-primary btn-lg px-4 me-md-2">Sign up</a>
-              <a asp-page="/Features" class="btn btn-outline-secondary btn-lg px-4">Learn more</a>
+    @if (User.Identity.IsAuthenticated)
+    {
+        if (Model.Spaces.Count == 0)
+        {
+            <div class="card">
+                <div class="card-header">
+                    Getting started
+                </div>
+                <div class="card-body">
+                    <h5 class="card-title">Welcome to the DLCS!</h5>
+                    <p class="card-text">As a new user, you can get started by creating a space to assemble content in.</p>
+                    <p>Here's a <a href="https://drive.google.com/file/d/14d-HcbftIt1qMhx-qV1jf29Hkn9e03E2/view">very short video</a> to get a feel for it.</p>
+                    <a asp-page="/Spaces/Index" class="btn btn-primary">Go to spaces</a>
+                </div>
             </div>
-          </div>
-        </div>
-      </div>
-    
-        
+        }
 
-        <div class="container my-5">
-            <div class="row p-4 pb-0 pe-lg-0 pt-lg-5 align-items-center rounded-3 border shadow-lg">
-                <div class="col-lg-7 p-3 p-lg-5 pt-lg-3">
-                    <h1 class="display-4 fw-bold lh-1">One asset?<br/>Or 100 million?</h1>
-                    <p class="lead">
-                        DLCS is a handy tool for ad-hoc IIIF production - just drag images in and create manifests straight away.
-                        But it also scales to collections with 100 million images, through its cost-effective tiered storage and
-                        delivery strategies.
-                    </p>
-                    <div class="d-grid gap-2 d-md-flex justify-content-md-start mb-4 mb-lg-3">
-              <a asp-page="/Signup" class="btn btn-primary btn-lg px-4 me-md-2">Sign up</a>
-              <a asp-page="/Features" class="btn btn-outline-secondary btn-lg px-4">Learn more</a>
+        <div class="container" id="icon-grid">
+            <div class="row row-cols-1 row-cols-sm-1 row-cols-md-3 row-cols-lg-3 g-4 py-5">
+                <!--
+                <div class="col d-flex align-items-start">
+                    <i class="text-muted flex-shrink-0 me-3" data-feather="list"></i>
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Queue/Index">Queue</a>
+                        </h4>
+                        <p>Paragraph of text beneath the heading to explain the heading.</p>
+                    </div>
+                </div>
+                -->

```

Comment:
commented out code

Reply:
Removed in [fc69e75](https://github.com/dlcs/protagonist/pull/640/commits/fc69e7553fd562d307aa1a6bc4b797d9a89a6b42)